modules/administration/pages/master-fingerprint.adoc

```diff
@@ -6,7 +6,7 @@ To set up validation from client to master enter the master's fingerprint within
 
 See the following procedure:
 
-. On the master enter the following command as root and note the fingerprint:
+. On the master enter the following command as root and note the fingerprint labeled as ``master.pub``:
```

Comment:
```suggestion
. On the master, at the command prompt, as root, use this command to find the ``master.pub`` fingerprint:
```

Reply:
@Loquacity lol you beat me to it, looks good!